src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindCollectionsIntegrationTest.java

```diff
@@ -236,7 +236,7 @@ public void systemKeyspace() {
           .contentType(ContentType.JSON)
           .body(json)
           .when()
-          .post(NamespaceResource.BASE_PATH, "system")
+          .post(NamespaceResource.BASE_PATH, "data_endpoint_auth")
```

Comment:
Add a comment line above saying something like "just need a keyspace that exists when database is created". Also change test method name to something that indicates what test does... like `checkNamespaceHasNoCollections()` or something.